packages/basic-modules/src/modules/font-size-family/menu/FontFamilyMenu.ts

```
@@ -48,14 +48,18 @@ class FontFamilyMenu extends BaseMenu {
+    let flag = false
 
     options.forEach(opt => {
       if (opt.value === curValue) {
+        flag = true
         opt.selected = true
       } else {
         delete opt.selected
       }
     })
+    // @ts-ignore
+    if (!flag) { options.push({ text: curValue, value: curValue, selected: true }) }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)